src/authentication/oauth2Auth.ts

```
@@ -259,8 +259,11 @@ export class Oauth2Auth extends AlfrescoApiClient {
     isPublicUrl(): boolean {
         const publicUrls = this.config.oauth2.publicUrls || [];
 
-        return publicUrls.length &&
-            publicUrls.some((urlPattern: string) => minimatch(window.location.href, urlPattern));
+        if (Array.isArray(publicUrls)) {
+            return publicUrls.length &&
+                publicUrls.some((urlPattern: string) => minimatch(window.location.href, urlPattern));
+        }
```

should we throw an error like in the case of others misconfigurations ?